### PR TITLE
Api docs complete

### PR DESCRIPTION
**Pull request by sjsrey**
- doctests passing
- api documentation
- code pruning

### CODE REVIEW

File: libpysal/cg/locators.py [hunk 1]
```diff
@@ -11,214 +11,10 @@
 from .standalone import *
 from .shapes import *
 
-__all__ = ["IntervalTree", "Grid", "BruteForcePointLocator",
+__all__ = ["Grid", "BruteForcePointLocator",
            "PointLocator", "PolygonLocator"]
 
 
-class IntervalTree:
```
Comment (ljwolf) [inline libpysal/cg/locators.py]: To make sure it's recorded somewhere, what's the logic behind removing interval tree?

File: doc/api.rst [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+.. autosummary::
+   :toctree: generated/
+
+   libpysal.weights.WSP
+   libpysal.weights.netW
+   libpysal.weights.mat2L
+   libpysal.weights.ODW
+   libpysal.weights.vecW
+   libpysal.weights.lat2W
```
Comment (weikang9009) [inline doc/api.rst]: `lat2W ` should be a Weights Util function?

File: doc/conf.py [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+
+    # Choose Bootstrap version.
+    # Values: "3" (default) or "2" (in quotes)
+    'bootstrap_version': "3",
+
+    'navbar_links': [
+                     #("Gallery", "auto_examples/index"),
+                     ("Installation", "installation"),
+                     ("API", "api"),
+                     ("References", "references"),
```
Comment (weikang9009) [inline doc/conf.py]: Should delete this line since there is not a references.rst

File: notebooks/voronoi.ipynb [hunk 4]
```diff
@@ -18,7 +18,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 24,
```
Comment (ljwolf) [inline notebooks/voronoi.ipynb]: I'm fine with merging it, but I think this suggests that the notebook was run twice without restarting.

Comment (sjsrey) [reply notebooks/voronoi.ipynb]: good catch. will fix before we do a release.

**Pull request merged**